nifi-pravega-processors/src/main/java/org/apache/nifi/processors/pravega/AbstractPravegaProcessor.java

```diff
@@ -144,10 +164,9 @@ public ValidationResult validate(String subject, String input, ValidationContext
     public ClientConfig getClientConfig(final ProcessContext context) {
         try {
             final URI controllerURI = new URI(context.getProperty(PROP_CONTROLLER).getValue());
-            final ClientConfig clientConfig = ClientConfig.builder()
-                    .controllerURI(controllerURI)
-                    .build();
-            return clientConfig;
+            final ClientConfig.ClientConfigBuilder clientBuilder = ClientConfig.builder().controllerURI(controllerURI);
+            clientBuilder.credentials(new PravegaKeycloakCredentialsFromString(context.getProperty(PROP_KEYCLOAK_JSON).getValue()));
```

Comment:
Need to handle case when Keycloak JSON is not provided.

Reply:
Good cache. Thanks
Implemented